bioblend/_tests/TestGalaxyJobs.py

```diff
@@ -0,0 +1,26 @@
+from . import (
+    GalaxyTestBase,
+    test_util
+)
+
+
+class TestGalaxyJobs(GalaxyTestBase.GalaxyTestBase):
+
+    @test_util.skip_unless_tool("random_lines1")
```

Comment:
```suggestion
    @test_util.skip_unless_galaxy('release_20.09')
    @test_util.skip_unless_tool("random_lines1")
```

Reply:
I think a normal rerun (which is what is being tested here) should be fine with older versions, right? It's only the remapping which won't work.

Reply:
The format of the state_inputs changed with 20.09, I don't think that works with older Galaxy versions.

Reply:
Okay, good to know.